src/plugins/usrbg/index.tsx

```diff
@@ -78,7 +82,7 @@ export default definePlugin({
 
     settingsAboutComponent: () => {
         return (
-            <Link href="https://github.com/AutumnVN/usrbg#how-to-request-your-own-usrbg-banner">CLICK HERE TO GET YOUR OWN BANNER</Link>
+            <Link href="https://github.com/AutumnVN/usrbg#how-to-request-your-own-usrbg-banner">How to request a banner</Link>
```

Comment:
why? need all caps because people are genuinely blind/dumb
```suggestion
            <Link href="https://github.com/AutumnVN/usrbg#how-to-request-your-own-usrbg-banner">CLICK HERE TO GET YOUR OWN BANNER</Link>
```

Reply:
Oh, it looked like YT clickbait to me which is why I changed it

Reply:
yes but the average USRBG user needs youtube clickbait to even find the button